### PR TITLE
Herm/feat/auth

### PR DESCRIPTION
**Pull request by hermsicle**

### CODE REVIEW

File: src/containers/_RegFormContainer.jsx [hunk 1]
```diff
@@ -53,33 +56,37 @@ const RegFormContainer = () => {
...
 							value={ email }
 							onChange={ ({ target }) => setEmail(target.value) }
 							error={ emailError ? true : false }
 						/>
-						{ emailError && <p>Email is required!</p> }
-					</RegForm.Frame>
-
-					<RegForm.Frame onClick={ () => setPasswordFocus(true) } >
 						<RegForm.Label
-							focus={ passwordFocus ? true : false }
```
Comment (awwmicky) [inline src/containers/_RegFormContainer.jsx:73]: I like how the ternary makes it explicit for true/false
the other can be just `passwordFocus ` (implicitly true/false)

File: src/containers/_RegFormContainer.jsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, useContext } from 'react'
 import { useHistory } from 'react-router-dom'
 import { RegForm } from '../components'
+import { EmailContext } from '../custom/EmailContext'
```
Comment (awwmicky) [inline src/containers/_RegFormContainer.jsx:4]: nice, starting to use context in the project

File: src/containers/_RegFormContainer.jsx [hunk 3]
```diff
@@ -53,33 +56,37 @@ const RegFormContainer = () => {
...
 							error={ emailError ? true : false }
 						/>
-						{ emailError && <p>Email is required!</p> }
-					</RegForm.Frame>
-
-					<RegForm.Frame onClick={ () => setPasswordFocus(true) } >
 						<RegForm.Label
-							focus={ passwordFocus ? true : false }
+							htmlFor="email"
 						>
```
Comment (awwmicky) [inline src/containers/_RegFormContainer.jsx:70]: I was wondering, what is your thought on the text for tags likes these?

```html
<!-- option 1 -->
<button
  title="click me"
>
  the click is real
</button>

<!-- option 2 -->
<button
  title="click me"
>the click is real
</button>

<!-- option 3 -->
<button
  title="click me"
>the click is real</button>
```

**Review by awwmicky — APPROVED** [review]
so far, everything is becoming better & better

File: src/containers/_RegFormContainer.jsx (hunk 3 above)
Comment (hermsicle) [reply src/containers/_RegFormContainer.jsx:70]: I personally like option 1.

**Pull request merged**